packages/components/src/button/style.scss

```diff
@@ -1,8 +1,10 @@
 // ==========================================================================
 // Buttons
 // ==========================================================================
+@import '../typography.scss';
 
 .button {
+	font-family: $aui-fontFamily-sans;
```

Comment:
The default fonts for `button` and `link` are different so we need to be specific.